index.html

```diff
@@ -0,0 +1,850 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/16d2250538.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>TNW</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo-flex flex">
+          <img
+            class="top-logo sec-1-img"
+            src="images/toplogo.png"
+            alt="Pic here"
+          />
+          <div class="float-left">
+            <div class="nav-1 flex">
+              <div class="nav-1-left">
+                <a href="#">News</a>
+                <a href="#">Events</a>
```

Comment:
#### Medium Screen Size

- [ ] The fonts in the middle nav are too different (they need to be bigger and centralized in the y-axis)

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91842723-eef47080-ec2a-11ea-9e83-0792821ee180.png)|![image](https://user-images.githubusercontent.com/61479648/91842781-03d10400-ec2b-11ea-8782-29ae2bf04f54.png) |

#### Mobile Screen Size

- [ ] The height of the layers is too different (the gray needs to be smaller and the yellow bigger)

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91842919-42ff5500-ec2b-11ea-93d0-256d292b8d9d.png) | ![image](https://user-images.githubusercontent.com/61479648/91842946-4bf02680-ec2b-11ea-8ba1-a4029395c985.png) |

- [ ] The layout breaks below 450px 

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91843090-835ed300-ec2b-11ea-8967-700c5cf09403.png) |![image](https://user-images.githubusercontent.com/61479648/91843123-92de1c00-ec2b-11ea-8568-999588fcc7b3.png)  |